w00_init/t01_big_picture.tex

```diff
@@ -247,7 +247,7 @@
 		\end{itemize}
 		\item Basics of \alert{Machine Learning} (mandatory)
 		\begin{itemize}
-			\item Classification, regression, clustering, decision tree, training-test split, cross validation, preprocessing \ldots
+			\item Classification, regression, clustering, decision tree, training-test split, cross validation, pre-processing \ldots
 			\item to catch up (if nec.): \url{https://www.coursera.org/learn/machine-learning} 
```

Comment:
You are right, preprocessing can be written that way. Writing "pre-processing" makes more sense when saying "pre- and post-processing" in my opinion.